sdk/communication/Azure.Communication.PhoneNumbers/src/CommunicationClientDiagnostics.cs

```diff
@@ -21,16 +25,12 @@ internal partial class ClientDiagnostics
         /// <param name="message">The error message.</param>
         /// <param name="errorCode">The error code.</param>
         /// <param name="additionalInfo">Additional error details.</param>
-#pragma warning disable CA1822 // Mark members as static
-        partial void ExtractFailureContent(
-#pragma warning restore CA1822 // Mark members as static
+        protected override void ExtractFailureContent(
```

Comment:
This implementation has extra "target" parsing. We are planning to tackle it but later.